torch/jit/__init__.py

```diff
@@ -804,3 +806,165 @@ def forward(self, *args, **kwargs):
 
 if not torch._C._jit_init():
     raise RuntimeError("JIT initialization failed")
+
+
+def _scalar(x):
+    """Convert a scalar tensor into a Python value."""
```

Comment:
This is pretty confusing, because we usually say "scalar tensor" when we mean a 0d tensor